### PR TITLE
Add loadModel, loadScene variants with import settings 

### PR DESCRIPTION
**Pull request by NolaDonato**
Adds the ability to specify import settings for loading scene and models
asynchronously. Unit test coming shortly.

GearVRF DCO signed off by: Nola Donato nola.donato@samsung.com

### CODE REVIEW

File: GVRf/Framework/framework/src/main/java/org/gearvrf/GVRAssetLoader.java [hunk 1]
```diff
@@ -1116,23 +1116,70 @@ public void loadScene(final GVRSceneObject model, final GVRResourceVolume volume
...
+     *          and other models contained within the model.
+     *          You can subclass GVRResourceVolume to provide custom IO.
+     * @param settings
+     *          Import settings controlling how assets are imported
+     * @param scene
+     *          Scene to be replaced with the model.
+     * @param handler
+     *          IAssetEvents handler to process asset loading events
+     */
+    public void loadScene(final GVRSceneObject model, final GVRResourceVolume volume, final EnumSet<GVRImportSettings> settings, final GVRScene scene, final IAssetEvents handler)
```
Comment (liaxim) [inline GVRf/Framework/framework/src/main/java/org/gearvrf/GVRAssetLoader.java]: Looks like the old ``loadScene(final GVRSceneObject model, final GVRResourceVolume volume, final GVRScene scene, final IAssetEvents handler)`` can easily be implemented by calling into the new loadScene and the duplication eliminated.

**Comment by liaxim** [comment]
Looks good. Planning to merge Wednesday morning.

File: GVRf/Framework/framework/src/main/java/org/gearvrf/GVRAssetLoader.java (hunk 1 above)
Comment (NolaDonato) [inline GVRf/Framework/framework/src/main/java/org/gearvrf/GVRAssetLoader.java]: I agree, I will make this change

**Comment by liaxim** [comment]
@NolaDonato This pull request now includes changes from the Vulkan branch. See the "merge vk_shader_rebased with master" commit.

**Comment by liaxim** [comment]
What should we do about this pull request?

**Comment by NolaDonato** [comment]
I will make a new pull request with just the 3 files that actually implement the feature

**Comment by NolaDonato** [comment]
replaced by #1243

**Pull request closed without merging**